### ISSUE TITLE
Visualization of Prediction Results

### ISSUE THREAD

**Issue by lhn352351688** [NUST-Machine-Intelligence-Laboratory/VideoMAC#4]
Hello, I am a beginner in this field. I used my own dataset (the training set consists of approximately 15,000 frames) to train the model, selected the tiny configuration, and trained for 200 epochs. However, when I used the trained model weights to reconstruct the test set images, the results were very poor. I suspect that my reconstruction code might be incorrect. Could you share your image reconstruction code? Thank you very much.

**Comment by PGSmall** [comment]
Please refer to [viz_rec](https://github.com/keyu-tian/SparK/blob/main/pretrain/viz_reconstruction.ipynb).
